cmd/node/config/config.toml

```diff
@@ -249,24 +249,26 @@
 [Antiflood]
     Enabled = true
     NumConcurrentResolverJobs = 50
-    PeerMaxMessagesPerSecond = 75
-    PeerMaxTotalSizePerSecond = 2097152
-    MaxMessagesPerSecond = 400
-    MaxTotalSizePerSecond = 4194304
+    [Antiflood.NetworkMaxInput]
```

Comment:
Nice split :)

Reply:
Yeah, became a mess there, split in more sub-settings as to be more coherent.